toc/rfc/accepted/rfc-draft-github-members.md

```diff
@@ -0,0 +1,36 @@
+# Meta
+[meta]: #meta
+- Name: Who is able to add and remove members of the Cloudfoundry GitHub org
+- Start Date: 2022-01-26
+- Author(s): @LeePorte
+- Status: Draft
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/194
+
+
+## Summary
+
+As the Cloudfoundry moves from the dojo model to an open committer model the existing central team in VMWare are no 
+longer appropriately positioned to grant and revoke access to users of the Cloudfoundry GitHub org
```

Comment:
I think it would be better for this section to summarize the proposed solution instead of the problem that is motivating the RFC.

Reply:
Good suggestion, I've modified the wording to reflect this.